app/controllers/api/v1/lookup_keys_controller.rb

```diff
@@ -66,6 +68,20 @@ def destroy
           process_response @lookup_key.destroy
         end
       end
+
+      private
+
+      def transform_default_value
+        default_value = params[:lookup_key].delete(:default_value)
+
+        default_value_attributes = {}
+        default_value_attributes.merge!({:value => default_value}) if default_value.present?
```

Comment:
You don't have to merge the hash, you can create it with the value straight ahead:

``` ruby
default_value_attributes = { :value => default_value } if default_value.present?
```